codex/rest/json.nim

```diff
@@ -55,6 +55,11 @@ func `%`*(arr: openArray[byte]): JsonNode =
 func `%`*(id: RequestId | SlotId | Nonce | AvailabilityId): JsonNode =
   % id.toArray
 
+func `%`*(obj: StorageRequest | Slot): JsonNode =
+  result = newJObject()
```

Comment:
I would not use the implicit result here (in fact, this is discouraged for various reasons and should be generally avoided https://status-im.github.io/nim-style-guide/language.result.html). Use an explicit `ret` or other variable and return that.

Reply:
Ok! Thanks for the feedback!

Reply:
Btw. there is quite some `result` usage in the codebase. Is there some Nim linter with which we could enforce rules like these?